ui/imports/shared/stores/TransactionStore.qml

```diff
@@ -20,7 +21,15 @@ QtObject {
     property var accounts: walletSectionAccounts.model
     property var currentAccount: walletSectionCurrent
     property string signingPhrase: walletSection.signingPhrase
-    property var savedAddressesModel: walletSectionSavedAddresses.model
+    property var savedAddressesModel: SortFilterProxyModel {
+        sourceModel: walletSectionSavedAddresses.model
+        filters: [
+            ValueFilter {
+                roleName: "isTest"
```

Comment:
Do you really want to see the "test" accounts only? Or the other way around?

Reply:
hey this is based on the variable are networksModule.areTestNetworksEnabled, so if test networks are enabled we will show the isTest=true saved addresses, if not enabled, we will not show the isTest=true saved addresses